python-avd/pyavd/_eos_designs/structured_config/network_services/router_bgp.py

```diff
@@ -191,6 +191,12 @@ def _router_bgp_vrfs(self: AvdStructuredConfigNetworkServices) -> dict:
                     # Resetting bgp_vrf so we only add global keys if there are any neighbors for VRF default
                     bgp_vrf = {}
 
+                if vrf_name == "default" and self.shared_utils.underlay_routing_protocol == "none":
```

Comment:
Could we move this up to line 180 with an `else` (as we know it is VRF default), then add a check for `/ if bgp_vrf is not empty and 2/ routing protocol==none

This saves one comparison in this if statement and groups everything happening for VRF default in one block (which we could move toa function

Reply:
done